### DISCUSSION TITLE
Using it on Android in termux.

### DISCUSSION THREAD

**Discussion by The-Thought-Magician** [aapatre/Automatic-Udemy-Course-Enroller-GET-PAID-UDEMY-COURSES-for-FREE#159] [General]
Hi fellas. I want to run this code on termux on my phone. I have installed python 3.9.1 on it. I know no coding so a bit stuck here. Please help me out.

**Comment by fakeid30** [comment]
Thanks for commenting here instead of opening an issue. We can not support mobile version of our program because it spins up a full blown browser instance. The webdriver we use does not provide selenium binary for mobile and it'll be too much of an effort to support it for this niche case (if at all possible).

You must use a computer operating system to run the script. Alternate will be connecting to a vps or windows rdp then invoking it remotely at which case the instructions will work verbatim. 

Let me know if you have any more question.

**Reply by The-Thought-Magician** [reply]
Can we share an account?

**Reply by fakeid30** [reply]
No we can't

**Reply by The-Thought-Magician** [reply]
Can you run the code in my account? If you got time🥺🥺🥺

**Reply by fakeid30** [reply]
no
